### PR TITLE
DYN-4685-Fix-PinButton-AddStyle

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

Update styles for the Pin Icon and AddStyle button.
I modified the style for the PinIcon for following the HIG standards also I modified the style for the "Add Style" button due that was not consistent with other buttons in the Preferences window. For the "Add Style" button I've added a Storyboard for modifying the Border.Background when the button is pressed.

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

Update styles for the Pin Icon and AddStyle button.

### Reviewers

@reddyashish 

### FYIs

@QilongTang

### CODE REVIEW

**Comment by QilongTang** [comment]
Snapshots of UI changes please, thanks!

**Comment by RobertGlobant20** [comment]
> Snapshots of UI changes please, thanks!

This is a GIF of the changes in the styles for the Pin button and AddStyle button.
styles_pinicon_addStyle

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml [hunk 1]
```diff
@@ -542,10 +542,8 @@
                                             <RowDefinition Height="Auto" />
                                         </Grid.RowDefinitions>
                                         <Button x:Name="AddStyleButton"
-                                                Style="{StaticResource FlatButtonStyle}" 
+                                                Style="{StaticResource SolidButtonStyleSmaller}"
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml:545]: Do we also need to apply this style for `ResetCPython` button?

Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml:545]: @QilongTang the `ResetCPython `button is using the style `SolidButtonStyle`, the style that I added SolidButtonStyleSmaller is almost the same that `SolidButtonStyle `the only changes are in a Border.Height, TextBlock.FontSize and the Background color triggered when the Button is disabled (based in the Figma design provided by Jingyi). I guess the FontSize/Height is smaller due that the AddStyle button is in a smaller space than the other buttons.
Here you can see a GIF of the two buttons
buttons_style

**Comment by QilongTang** [comment]
One comment then LGTM

File: src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml (hunk 1 above)
Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml:545]: @QilongTang comment added in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12738/commits/180c9b1dade33077256f9278733cf40491bd6e7f

**Review by QilongTang — APPROVED** [review]

**Pull request merged**